adjust upgrade doc (#23)
OpenFunction/charts: README.md

### From OpenFunction v0.6.0 to OpenFunction v0.7.x

> There is an incompatible breaking change needing manual actions.

Reviewer: ```suggestion
There is a breaking change when upgrading from v0.6.0 to 0.7.x which requires additional manual operations.
```

#### Uninstall the Chart

> To uninstall/delete the `openfunction` release:

Reviewer: ```suggestion
First, you'll need to uninstall the old `openfunction` release:
```

```shell
helm uninstall openfunction -n openfunction
```

Confirm that the component namespaces have been deleted, it will take a while:
```shell
kubectl get ns -o=jsonpath='{range .items[?(@.metadata.annotations.meta\.helm\.sh/release-name=="openfunction")]}{.metadata.name}: {.status.phase}{"\n"}{end}'
```

> If the knative-serving namespace is in the terminating state for a long time, try running the following command and remove finalizers:
```shell
kubectl edit ingresses.networking.internal.knative.dev -n knative-serving
```

> #### Upgrade OpenFunction CRDs

Reviewer: ```suggestion
#### Upgrade OpenFunction CRDs
Then you'll need to upgrade the new OpenFunction CRDs
```

```shell
kubectl apply -f https://openfunction.sh1a.qingstor.com/crds/v0.7.0/openfunction.yaml
```

> #### Upgrade components CRDs

Reviewer: ```suggestion
#### Upgrade dependent components CRDs
You also need to upgrade the dependent components' CRDs
```

    kubectl apply -f https://openfunction.sh1a.qingstor.com/crds/v0.7.0/tekton-pipelines.yaml
    ```

> #### Install new chart

Reviewer: Simply installing the new chart will not upgrade all the CRDs?

Author: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations

https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md#upgrading-chart

CRDs can be managed in templates , keda and contour both manage CRDs this way, but then `helm uninstall` will delete all custom resources(for us is Function), dapr does not manage CRDs this way. Also Knative's webhook blocks the patch operation performed by helm upgrade ，so currently CRDs are not managed in this way.
